### PR TITLE
Be able to add migrations and add migration for footyPoints

### PR DESCRIPTION
**Pull request by sbetti02**
This should make a more painless process to add migrations for schema changes, using the migrate-mongo command.

I've also added a short script that can be used for running deploys.  This just automates the process of what had been 5-6 manual steps, but also adds in the `migrate-mongo up` command to deploy any new migrations as part of the release.

### CODE REVIEW

File: server/migrate-mongo-config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// In this file you can configure migrate-mongo
+
+const config = {
+  mongodb: {
+    // TODO: Update this with prod config
+    url: "mongodb://localhost:27017",
```
Comment (reshadbinharun) [inline server/migrate-mongo-config.js:6]: Would these not use `process.env` to fetch the url and dbname?

File: server/migrate-mongo-config.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    // TODO: Update this with prod db name
+    databaseName: "ofi-testdata",
+
+    options: {
+      useNewUrlParser: true, // removes a deprecation warning when connecting
+      useUnifiedTopology: true, // removes a deprecating warning when connecting
+      //   connectTimeoutMS: 3600000, // increase connection timeout to 1 hour
```
Comment (reshadbinharun) [inline server/migrate-mongo-config.js:14]: We could get rid of these commented out config params right?

**Review by reshadbinharun — CHANGES_REQUESTED** [review]
Looking solid, 2 comments!

File: server/migrate-mongo-config.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+// In this file you can configure migrate-mongo
+require('dotenv').config();
+console.log(process.env.DB)
+
+const config = {
+  mongodb: {
+    // TODO: Update this with prod config
+
```
Comment (reshadbinharun) [inline server/migrate-mongo-config.js:8]: nit: stray lines of whitespace

File: server/migrate-mongo-config.js (hunk 2 above)
Comment (reshadbinharun) [reply server/migrate-mongo-config.js:18]: bump on this comment!

File: server/migrate-mongo-config.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+// In this file you can configure migrate-mongo
+require('dotenv').config();
+console.log(process.env.DB)
```
Comment (reshadbinharun) [inline server/migrate-mongo-config.js:3]: remove log statement

File: server/migrate-mongo-config.js (hunk 2 above)
Comment (sbetti02) [reply server/migrate-mongo-config.js:18]: My bad, missed these comments, will take care of them now

**Review by reshadbinharun — APPROVED** [review]
Let's GOOOOOO! 🚀

**Pull request merged**